modules/nf-core/shinyngs/validatefomcomponents/main.nf

```diff
@@ -2,18 +2,21 @@ process SHINYNGS_VALIDATEFOMCOMPONENTS {
     tag "$sample"
     label 'process_single'
 
-    conda "bioconda::r-shinyngs=1.5.1"
+    conda "bioconda::r-shinyngs=1.5.4"
```

Comment:
Sorry, one last thing- would you be able to bump the other shinyngs modules ass well (exploratory, differential)?

Then this PR will also close https://github.com/nf-core/differentialabundance/issues/56.